### PR TITLE
Document that duration out of range can't be reached for /extend

### LINKED ISSUE #377: How to reach `DURATION_OUT_OF_RANGE` for `/sessions/<sessionId>/extend`?
**Describe the bug**

>  Extend the overall session duration of an active QoS session (with qosStatus = AVAILABLE).
        The overall duration of the QoS session, including the additional extended duration, shall not exceed the maximum duration limit fixed for the QoS Profile. If the current duration plus the value of `requestedAdditionalDuration` exceeds the maximum limit, the new overall duration shall be capped to the maximum value allowed.
        An example: For a QoS profile limited to a `maxDuration` of 50,000 seconds, a QoD session was originally created with duration 30,000 seconds. Before the session expires, the developer requests to extend the session by another 30,000 seconds:
        - Previous duration: 30,000 seconds
        - Requested additional duration: 30,000 seconds
        - New overall session duration: 50,000 seconds (the maximum allowed)
https://github.com/camaraproject/QualityOnDemand/blob/main/code/API_definitions/quality-on-demand.yaml#L339-L344

Let's take this example here.
So after the extension, the overall session duration was set to 50.000 / the maximum of the QoS-profile.
What would happen, if now the user requests for another extension of 30.000 seconds, right after the previous extension? 

a) The overall session will again be set to 50.000
b) The provider will answer with a HTTP - 400 and `QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE` (see https://github.com/camaraproject/QualityOnDemand/blob/main/code/API_definitions/quality-on-demand.yaml#L1053-L1058)
or c) The provider just allows extension until the maximum is reached, afterwards the session cannot be extended anymore.

If a) is the right answer, and you can extend as much as you want, then I guess `QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE` is just a valid error for first-time creation of a session. So not anymore for the extension.

Wdyt?

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

Add one of the following kinds:

* documentation

#### What this PR does / why we need it:

Documenting within the API definition of v0.11.x that the error response QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE will be deleted for the operation /extend within the next API version.

#### Which issue(s) this PR fixes:

Fixes #377

#### Special notes for reviewers:

Note: A new issue must be created for the final removal of the error response within the next release cycle.

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

File: code/API_definitions/quality-on-demand.yaml [hunk 1]
```diff
@@ -1051,7 +1051,7 @@ components:
                 code: OUT_OF_RANGE
                 message: Client specified an invalid range.
             DurationOutOfRangeForQoSProfile:
-              description: The requested duration is out of the allowed range for the specific QoS profile
+              description: NOTE - this error response can't be reached within the `/extend` operation and will be removed from the API definition within the next version.
```
Comment (eric-murray) [inline code/API_definitions/quality-on-demand.yaml:1054]:
```suggestion
              description: NOTE - this error response can't be reached within the `/extend` operation and will be removed from the API definition within the next version.
```

Comment (hdamker) [reply code/API_definitions/quality-on-demand.yaml]: Thanks

**Review by jlurien — APPROVED** [review]
Minor "formal" suggestion

File: code/API_definitions/quality-on-demand.yaml [hunk 2]
```diff
@@ -1060,7 +1060,7 @@ components:
                 code: OUT_OF_RANGE
                 message: Client specified an invalid range.
             DurationOutOfRangeForQoSProfile:
-              description: The requested duration is out of the allowed range for the specific QoS profile
+              description: NOTE - this error response cannot be reached within the `/extend` operation and will be removed from the API definition within the next version.
```
Comment (eric-murray) [inline code/API_definitions/quality-on-demand.yaml:1063]:
```suggestion
              description: NOTE - this error response cannot be returned by the `/extend` operation and will be removed from the API definition in the next version.
```

**Review by eric-murray — APPROVED** [review]

**Pull request merged**